pkg/policy/api/utils.go

```diff
@@ -65,6 +65,7 @@ func (k *PortRuleKafka) Exists(rules L7Rules) bool {
 
 // Equal returns true if both rules are equal
 func (k *PortRuleKafka) Equal(o PortRuleKafka) bool {
+	//TODO
```

Comment:
leftover? If no, please add a more descriptive message and create GH

Reply:
I think you are seeing an older commit. this was cleaned up

Reply:
fixed